src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -59,4 +68,12 @@ public Integer getRecommendedLength() {
     public void setRecommendedLength(Integer recommendedLength) {
         this.recommendedLength = recommendedLength;
     }
+
+    public boolean getShouldAllowRichText() {
+        return shouldAllowRichText;
+    }
+
+    public void setShouldAllowRichText(boolean shouldAllowRichText) {
```

Comment:
Since your variable is a boxed type instead of a primitive, would recommend following the same convention here (i.e. `Boolean` instead of `boolean`. Unless you've tried that and it doesn't work?

Reply:
So sorry for the late reply too, just ended my semester with some leftover matters taken care of too. I have not tried it, but it must be an oversight from me that this remains as unboxed. Will change it.